### PR TITLE
ISSD-3003 [PRM] Add badge error for Activities list in MDF Request form

### PR DESCRIPTION
**Pull request by eliasantosf**
Task: [ISSD-3003](https://liferay.atlassian.net/browse/ISSD-3003)

### CODE REVIEW

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestForm/components/ActivityPanel/ActivityPanel.tsx [hunk 1]
```diff
@@ -34,71 +35,82 @@ const ActivityPanel = ({
...
-		>
-			{detail && <ClayPanel.Body>{children}</ClayPanel.Body>}
-		</ClayPanel>
+			{hasErrors && (
+				<div>
+					<ClayAlert displayType="danger" hideCloseIcon={true}>
+						{`Missing information! Please complete the activity's required
+					fields before proceeding to the next step.`}
+					</ClayAlert>
+				</div>
```
Comment (jairmedeiros) [inline workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestForm/components/ActivityPanel/ActivityPanel.tsx:111]:
```suggestion
					<ClayAlert displayType="danger" hideCloseIcon={true}>
						Missing information! Please complete the activity's required
					fields before proceeding to the next step.
					</ClayAlert>
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestForm/components/ActivityPanel/ActivityPanel.tsx (hunk 1 above)
Comment (eliasantosf) [reply workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestForm/components/ActivityPanel/ActivityPanel.tsx]: Done!

**Comment by jairmedeiros** [comment]
ci:forward:force

**Comment by jairmedeiros** [comment]
ci:forward:force

**Pull request closed without merging**